termui/show_bug.go

```diff
@@ -94,7 +94,7 @@ func (sb *showBug) layout(g *gocui.Gui) error {
 	}
 
 	v.Clear()
-	fmt.Fprintf(v, "[q] Save and return [←↓↑→,hjkl] Navigation ")
+	fmt.Fprintf(v, "[q] Save and return [←↓↑→,hjkl] Navigation [o] Toggle open/close ")
```

Comment:
To avoid displaying too much instructions at the same time, it would be best to display this one only when the cursor is on the bug timeline. Could you move it a few line below, with the instructions for comments/title ?